index.html

```diff
@@ -20,12 +20,152 @@
+        </section>
+        <section class="desk_projects-container2">
+            <div class="projectpageimage3"></div>
+            <img class="newimg_grid2" src="Shape_thirdgrid.svg" alt="gridimage">
+            <img class="newimg1_grid2" src="Group_83.svg" alt="gridimage">
+        </section>
+        <section class="desk_projects-container3">
+            <img class="newimg_grid3" src="Group_85.svg" alt="gridimage">
+            <img class="newimg1_grid3" src="image-geometry_4.svg" alt="gridimage">
+            <img class="newimg2_grid3" src="Shape_grid3.svg" alt="gridimage">
+            <div class="projectpageimage4"></div>
+        </section>
+        <section class="desk_languages-container">
+            <div class="Rectangle11_grid3">
+                <div class="Languages_test">Languages</div>
+                <img class="img6" src="hash.png" alt="Hash symbol">
+                <ul id="navigat_grid" class="list_items_languages">
+                    <li class="jslist">Javascript (ES5/ES6)</li>
+                    <li class="rubylist">Ruby</li>
+                    <li class="htmllist">HTML</li>
+                    <li class="csslist">CSS</li>
+                </ul>
+            </div>
+            <div class="Rectangle12_grid3">
+                <div class="Frameworks_test">Frameworks</div>
+                <img class="img8" src="shape2.png" alt="Hash symbol">
+                <ul id="navigat1_grid" class="list_items_languages">
+                    <li class="bootstraplist">Bootstrap</li>
+                    <li class="rubyraillist">Ruby on rails</li>
+                    <li class="rspeclist">RSpec</li>
+                    <li class="caplist">Capibara</li>
+                    <li class="sellist">Selenium</li>
+                </ul>
+            </div>
+            <div class="Rectangle13_grid3">
+                <div class="Skills_test">Skills</div>
+                <img class="img10" src="shape3.png" alt="Hash symbol">
+                <ul id="navigat2_grid" class="list_items_languages">
+                    <li class="codekitlist">Codekit</li>
+                    <li class="codepenlist">Codepen</li>
+                    <li class="githublist">GitHub</li>
+                    <li class="gitlablist">GitLAb</li>
+                    <li class="termlist">Terminal</li>
+                </ul>
+            </div>
+            <img class="newimg_grid33" src="Group_84.svg" alt="gridimage">
+            <img class="newimg1_grid33" src="Shape_grid33.svg" alt="gridimage">
+            <a name="abouttext">
+                <div class="abouttext">About</div>
+            </a>
+            <div class="aboutpara">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</div>
+            <div class="rectangle14_grid33">
+                <div class="getmycv">Get my CV</div>
```

Comment:
This is great you followed the design here and it looks amazing. 

- [ ]  Make sure it's a button.

Reply:
I have changed it to the button as per your suggestion.